Makefile

```diff
@@ -62,8 +63,10 @@ tidy:
 
 format:
 	@echo ">> formatting code"
-	@$(GO) fmt $(pkgs)
-	@cd cmd && $(GO) fmt $(cmd_pkgs)
+	@# Can't use $(pkgs) and $(cmd_pkgs) here as we need
+	@# to skip auto-generated files.
+	@find -name '*.go' | grep -Ev '^vendor/|.pb.go$$' | \
+		xargs goimports -w -local github.com/google/cadvisor
```

Comment:
Can we leave a comment that `goimports -w` is more or less `gofmt` :)

Reply:
done